Tests/priority_queue_test.py

```diff
@@ -16,12 +16,13 @@ def test_insertion(self):
     def test_proper_order(self):
         """Test that a mixed-bag of priorities comes out in the right order."""
         queue = PriorityQueue()
-        items = [1, 2, 3]
-        priorities = [Priorities.LOW, Priorities.MEDIUM, Priorities.HIGH]
+        items = [1, 2, 3, 4]
+        priorities = [Priorities.LOW, Priorities.LOW, Priorities.MEDIUM,
```

Comment:
theres 2 LOWs here

Reply:
That's the reason for the pull request. There are going to be situations where we may put a bunch of instructions on the queue that are of equal priority. We want instructions of equal priority to be read in the order they were put in the queue. The old implementation had it so that the most recent instruction (of equal priority) to be added to the queue was executed first.

Reply:
How does duplicating the low priority help with that

Reply:
In the test, the item with value 1 should be dequeued before the item with value 2, since they are equal priority (LOW) and the item with value 1 is enqueued first.

Reply:
Since this is a test function, it's perfectly reasonable for two lows to be in the list here as it verifies that item priority is handled appropriately, and even if it were an erroneous case, then that's still something valid to test for.